src/main/java/com/kainos/ea/models/Job.java

```diff
@@ -17,14 +17,23 @@ public class Job {
     private String name;
     @NotEmpty
     private String job_description;
+    @NotEmpty
+    private String job_responsibilities;
+
+    public Job() {
+    }
 
     public Job(int id, String name, String job_description) {
         this.id = id;
         this.name = name;
         this.job_description = job_description;
     }
 
-    public Job() {
+    public Job(int id, String name, String job_description, String job_responsibilities) {
```

Comment:
Can drop the prefix here too, we're in a job class. `Job.responsibilities` reads nicer than `Job.job_responsibilities`

Reply:
Sure thing, we've made the changes for all active feature branches due to the global nature of changes to the SQL table structure.

Reply:
Okay, arguably though that's what merging is for - so changes don't need duplicated across feature branches.